controller/users.go

```diff
@@ -266,6 +253,17 @@ func (c *UsersController) createUserInDB(ctx *app.CreateUsersContext, identityID
 		if err != nil {
 			return err
 		}
+
```

Comment:
What about refactoring a little more this controller and move the logic of creating the user and identity, as well as calling the remote `WIT` and `Cluster` services in the `UserService`? 
AFAIU, this is the design that we should use: controllers validate the input then call services which are responsible for carrying the transaction and the dealing with the business logic. In this case, the call to the local repositories and remote services would take place within the same transaction, as you already did here.

Reply:
1. We can't call remote service inside DB transactions.
2. Let's not blow up this PR by starting refactoring not related code. This controller requires refactoring indeed but let's do it as a separate task. We have to move forward with more important tasks right now.